### ISSUE TITLE
FR: Copy Markdown Link to Note

### ISSUE THREAD

**Issue by brimwats** [chrisgrieser/obsidian-smarter-md-hotkeys#35]
### Feature Requested

I can add markdown links to my task manager. I think the ability to copy a note URI in markdown formatting would complement the copy file name and copy relative/absolute path. A link would look like:

[Quals Meeting With Julia](obsidian://advanced-uri?vault=booxsidian&filepath=Quals%20Meeting%20w%20Julia.md&heading=Notes)

I don't see another plugin that does this

### Relevant Screenshot

![image](https://user-images.githubusercontent.com/73113656/181003325-77f3b7f0-c9a6-4382-9f66-7b942daf406e.png)

the commands that this would fit next to nicely

### Which editor are you using?

- [X] New Editor
- [ ] Legacy Editor

### Checklist

- [X] I have checked that there isn't [another hotkey plugin](https://publish.obsidian.md/hub/02+-+Community+Expansions/02.01+Plugins+by+Category/Plugins+that+add+or+manage+hotkeys) that does what I want.
- [X] The feature would be useful to more users than just me.

**Comment by chrisgrieser** [comment]
You mean something like [Meeting with Julia](obsidian://...)? That would be doable

---
*note to myself when I find the time to implement this*
the undocumented API `app.getObsidianUrl(file: TFile)` can be used for this.

**Comment by brimwats** [comment]
yeah the final link would be like [Meeting with Julia](obsidian://.......)
